step-capstone/src/components/TripForms/TripSettingFormPopover.js

```diff
@@ -128,17 +140,15 @@ function EditTripSetting(props) {
                         onChange={handleTitleChange}
                     />
                 </Grid>
-
                 <Grid item container direction="row" justify="space-between"></Grid>
                 <LocationAutocompleteInput
-                    onLocationSelected={handleDestinationsChange}
-                    error={destinations === ""}
-                    helperText={(destinations === "") ? "Cannot leave field blank, list your destinations, and separate them using comma" : null}
-                    text={destinations}
+                    onLocationSelected={handleDestinationChange}
+                    error={!destination}
+                    helperText={!destination ? "Cannot leave field blank, list your destinations, and separate them using comma" : null}
```

Comment:
we should change this helperText since we are not supporting multiple destinations

Reply:
Nice catch